evaluations/data/evaluation_configs.py

```diff
@@ -77,10 +77,131 @@ def _smoke_test(num_runs=100):
           )
       )
 
+
+def _test_with_selected_parameters_second_part(num_runs=100):
```

Comment:
Does it make sense to have unit tests for this code?  Personally, I try to avoid checking in any code unless there is some kind of test for it, even a rudimentary one.

Reply:
Thanks Matthew. Per discussion with Xichen, plan to:
(1) Test whether this evaluator can be run through, with just 1 run.
(2) Test the generated set_size_list for small num_sets.

Reply:
Done.